index.js

```diff
@@ -6,18 +6,18 @@ module.exports = {
 		dashicon: 'palmtree',
 		version: '1.0.0',
 		npmDependencies: [
-			"@wordpress/icons@^9.22.0",
+			"@wordpress/icons@^10.13.0",
```

Comment:
```suggestion
			"@wordpress/icons@^10.14.0",
```

Nitpic: A few hours ago, version [`10.14.0`](https://www.npmjs.com/package/@wordpress/icons) has been released.